core/src/main/scala/JwtUtils.scala

```diff
@@ -13,6 +13,8 @@ object JwtUtils {
   val ENCODING = "UTF-8"
   val RSA = "RSA"
   val ECDSA = "EC"
+  val EdDSA = "EdDSA"
+  val Ed25519 = "Ed25519"
```

Comment:
As `val Ed25519` is only used in the tests, are you OK to move it to `Fixture.scala`?

This way here we only have constants for the "families" of algorithms to be passed to the parsers, not the specific algorithms (just like we don't have constants here for `RS256`, `ES512`, etc).